cli/rbd/trash/trash.py

```diff
@@ -0,0 +1,240 @@
+from copy import deepcopy
+
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+
+# from .trash_purge_schedule import TrashPurgeSchedule
```

Comment:
please remove this?

Reply:
This will be required in my subsequent PRs for trash. like I said this is just a partial PR. So I would like to retain this here for now. So I don't miss out on adding it in the next PR